lib/flutter_secure_storage.dart

```diff
@@ -141,12 +141,16 @@ enum IOSAccessibility {
 class IOSOptions extends Options {
   IOSOptions(
       {String groupId,
-      IOSAccessibility accessibility = IOSAccessibility.unlocked})
+      IOSAccessibility accessibility = IOSAccessibility.unlocked,
+      bool useFlutterSecureStorageServiceAsAccountName = true,
```

Comment:
@klyver 
what do you think about  passing AccountName as String with default value `flutter_secure_storage_service`?

Reply:
That will probably be better. Then `IOSOptions(accountName: null)` should be allowed and be the way to do it when the user does not want kSecAttrService to be set at all.